torch_xla/csrc/tensor_methods.cpp

```diff
@@ -2506,6 +2506,20 @@ std::tuple<XLATensorPtr, XLATensorPtr> topk(const XLATensorPtr& input,
       input->CreateFrom(torch::lazy::Value(node, 1), at::ScalarType::Long));
 }
 
+std::tuple<XLATensorPtr, XLATensorPtr> topk_symint(const XLATensorPtr& input,
```

Comment:
quick question, now that we have a symint version of `topk`, do we still need the original `topk`?

Reply:
I basically followed the `expand_symint` design, which keeps the static expand implementation. I guess the original `topk` implementation is still needed because the codegen for handling SymInt is slightly different (e.g. SetDimensionSize).

Reply:
yea, let's keep the static version their.